src/ConfiguredMediator.php

```diff
@@ -37,15 +44,43 @@ public function uninstall(Composer $composer, IOInterface $io)
 
     public function installOrUpdateFunction(PackageEvent $event): void
     {
+        $gnuPG = $this->createGnuPG();
+        // we do not want to crash if no GnuPG was found
+        // but display a noticeable warning to the user
+        if (null === $gnuPG && !self::$warningDisplayed) {
```

Comment:
No yoda :-)

Reply:
Switched all Yoda conditionals to regular ones